### ISSUE TITLE
在使用autostart设置开机自启的时候，好像没有生效

### ISSUE THREAD

**Issue by Sunrisies** [tauri-apps/plugins-workspace#1922]
![image](https://github.com/user-attachments/assets/a9c2ea53-9cbe-4fbb-8ace-a6aadc7848f2)
![image](https://github.com/user-attachments/assets/993e193f-fc82-44d4-a835-813eeac63dd6)
![image](https://github.com/user-attachments/assets/0d154810-26fe-40b0-8fc4-8ebadd018831)

在使用这个的时候开启自启好像不生效，是我使用方法不对吗？

**Comment by FabianLars** [comment]
Can you also share the output of the `tauri info` command please?

**Comment by Sunrisies** [comment]
> Can you also share the output of the `tauri info` command please?

![image](https://github.com/user-attachments/assets/7a125faa-4c93-44f9-87f2-6f71634a6c60)
是这个吗？

**Comment by FabianLars** [comment]
yes that's it, thanks. Would it be possible to update all the tauri dependencies to stable v2?

**Comment by Sunrisies** [comment]
![image](https://github.com/user-attachments/assets/23630e3c-b357-480a-8f01-5dd86247985e)


> yes that's it, thanks. Would it be possible to update all the tauri dependencies to stable v2?是的，就是这样，谢谢。是否可以将所有 tauri 依赖项更新到稳定的 v2？

这个是我目前的依赖

**Comment by FabianLars** [comment]
I think the `tauri migrate` command can only upgrade from beta, not rc. But really, there shouldn't be any big changes from the last rc versions to stable so simply removing the `-rc.X` suffix from all the tauri crates there should just work. (same in package.json)

**Comment by Sunrisies** [comment]
![image](https://github.com/user-attachments/assets/4c008765-87fd-49df-b509-69808881a966)
还是不生效

**Comment by Xiaobaishushu25** [comment]
一样的不生效，请问你解决了吗

**Comment by Sunrisies** [comment]
> 一样的不生效，请问你解决了吗

并没有

**Comment by Ca11MeE** [comment]
> > 一样的不生效，请问你解决了吗
> 
> 并没有

我也是不生效，任务管理器能注册上去，重启没有运行程序。感觉跟我在build.rs增加管理员授权有关系。打算换auto_launch试试

**Comment by ayangweb** [comment]
Some users have reported that the feature isn’t working, but I’m not sure where to start investigating the issue.

**Comment by ayangweb** [comment]
This issue is quite strange — it only occurs for a very small number of users (the toggle is indeed enabled, and they’ve also tried turning it off and on again), while it works fine for most others.

**Comment by XuGuangnian** [comment]
The dependency auto-launch 0.5 does not support automatically restarting applications that run with administrator privileges. Upgrading to auto-launch 0.6 (released last month) should be able to resolve this issue.

**Comment by ayangweb** [comment]
@FabianLars Hey👋, according to this comment ([https://github.com/tauri-apps/plugins-workspace/issues/1922#issuecomment-3858495164](https://github.com/tauri-apps/plugins-workspace/issues/1922#issuecomment-3858495164)), should we bump the version?

**Comment by FabianLars** [comment]
it's incompatible with our msrv policy

**Comment by Legend-Master** [comment]
If anyone's interested, you can open PRs upstream https://github.com/zzzgydi/auto-launch to request a change on `windows-registry = "^0.6"` to `windows-registry = ">=0.5, <=0.6"` to lower the MSRV so we can update it here (and ask him to publish a new version)

Also it seems like they have added `os_info` but only used on macOS, it would be nice to clean that up as well
